### PR TITLE
[PATCH] update to v1.8.0

### PR DESCRIPTION
**Pull request by SquaredCanine**
Added functionality described in the changelog. Plus minor improvements of the different skills.

### CODE REVIEW

File: Presentation/src/main/kotlin/furhatos/app/presentation/flow/presentation.kt [hunk 1]
```diff
@@ -16,42 +53,102 @@ val Start : State = state {
...
+val Ending : State = state{
+    onEntry {
         furhat.say("These are exciting times")
         delay(100)
         attendRandomUserOrLocation()
 
-        furhat.say("I believe that great things are about to happen between social robots and humans")
-        delay(200)
+
+        val myOtherUtterance = "between social robots and humans"
```
Comment (luddilo) [inline Presentation/src/main/kotlin/furhatos/app/presentation/flow/presentation.kt]: Please make this an utterance

File: Presentation/src/main/kotlin/furhatos/app/presentation/flow/presentation.kt [hunk 2]
```diff
@@ -16,42 +53,102 @@ val Start : State = state {
...
+    }
+}
 
+val DifferentPersonalities : State = state {
+    onEntry {
         furhat.say("I can have different personalities")
         furhat.setTexture("female")
         furhat.say("I can look like a woman")
-        furhat.setVoice(Language.ENGLISH_US, Gender.FEMALE)
+        furhat.setVoice(globalDefaultFemale.first, globalDefaultFemale.second)
```
Comment (luddilo) [inline Presentation/src/main/kotlin/furhatos/app/presentation/flow/presentation.kt]: Instead use named voices

File: Presentation/src/main/kotlin/furhatos/app/presentation/flow/wizardButtons.kt [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package furhatos.app.presentation.flow
+
+import furhatos.flow.kotlin.Color
+import furhatos.flow.kotlin.Section
+import furhatos.flow.kotlin.partialState
+
+val wizardButtons  = partialState {
+
+    //If the spacebar is pressed, the presentation is started
+    onButton(key=" ") {
```
Comment (luddilo) [inline Presentation/src/main/kotlin/furhatos/app/presentation/flow/wizardButtons.kt]: Hidden please, add space.

File: Quiz/src/main/kotlin/furhatos/app/quiz/questionlist.kt [hunk 4]
```diff
@@ -0,0 +1,89 @@
+package furhatos.app.quiz
+
+/**
+ * The questions are structured like
+ *  -The question
+ *  -The correct answer, followed by alternative pronounciations
+ *  -A list of other answers followed by their alternatives
+ */
```
Comment (luddilo) [inline Quiz/src/main/kotlin/furhatos/app/quiz/questionlist.kt]: Should we used named parameters here to make this easier to read?

File: WolframAlpha/src/main/kotlin/furhatos/app/wolframalpha/flow/general.kt [hunk 5]
```diff
@@ -5,7 +5,8 @@ import furhatos.util.*
 
 val idle: State = state {
     init {
-        furhat.setVoice(Language.ENGLISH_US, Gender.MALE)
+        furhat.setTexture("male")
+        furhat.setVoice(Language.ENGLISH_US, "Matthew")
```
Comment (luddilo) [inline WolframAlpha/src/main/kotlin/furhatos/app/wolframalpha/flow/general.kt]: Same as above

File: Presentation/src/main/kotlin/furhatos/app/presentation/flow/presentation.kt (hunk 1 above)
Comment (SquaredCanine) [reply Presentation/src/main/kotlin/furhatos/app/presentation/flow/presentation.kt]: Changed!

File: Presentation/src/main/kotlin/furhatos/app/presentation/flow/presentation.kt (hunk 2 above)
Comment (SquaredCanine) [reply Presentation/src/main/kotlin/furhatos/app/presentation/flow/presentation.kt]: Changed!

File: Presentation/src/main/kotlin/furhatos/app/presentation/flow/wizardButtons.kt (hunk 3 above)
Comment (SquaredCanine) [reply Presentation/src/main/kotlin/furhatos/app/presentation/flow/wizardButtons.kt]: Changed!

File: Quiz/src/main/kotlin/furhatos/app/quiz/questionlist.kt (hunk 4 above)
Comment (SquaredCanine) [reply Quiz/src/main/kotlin/furhatos/app/quiz/questionlist.kt]: Named!

File: WolframAlpha/src/main/kotlin/furhatos/app/wolframalpha/flow/general.kt (hunk 5 above)
Comment (SquaredCanine) [reply WolframAlpha/src/main/kotlin/furhatos/app/wolframalpha/flow/general.kt]: Changed!

**Pull request merged**